### PR TITLE
chore: no std as in other mods for seeded

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -45,10 +45,13 @@ pub type FxHashMap<K, V> = HashMap<K, V, BuildHasherDefault<FxHasher>>;
 #[cfg(feature = "std")]
 pub type FxHashSet<V> = HashSet<V, BuildHasherDefault<FxHasher>>;
 
-#[cfg(feature = "rand")]
+#[cfg(all(feature = "rand", feature = "std"))]
```
Comment (Nilstrieb) [inline src/lib.rs:48]: Same here

**Comment by vados-cosmonic** [comment]
This is now a duplicate of https://github.com/rust-lang/rustc-hash/pull/41

File: src/seeded_state.rs [hunk 2]
```diff
@@ -41,9 +44,11 @@ impl core::hash::BuildHasher for FxSeededState {
 mod tests {
     use core::hash::BuildHasher;
 
+    #[cfg(feature = "std")]
     use crate::{FxHashMapSeed, FxSeededState};
 
     #[test]
+    #[cfg(feature = "std")]
```
Comment (WaffleLapkin) [inline src/seeded_state.rs:51]: `#[test]` requires `std`, is this gating actually useful?

**Comment by dzmitry-lahoda** [comment]
thank you.

**Pull request closed without merging**